Examples/Java/Sources/Board.java

```diff
@@ -0,0 +1,56 @@
+//
+// Board.java
+// Autogenerated by plank
+//
+// DO NOT EDIT - EDITS WILL BE OVERWRITTEN
+// @generated
+//
+
+package com.pinterest.models;
+
+import android.support.annotation.IntDef;
+import com.google.auto.value.AutoValue;
+import java.net.URI;
+import java.util.Set;
+import android.support.annotation.Nullable;
+import android.support.annotation.StringDef;
+import java.lang.annotation.Retention;
+import java.util.Date;
+import java.util.List;
+import java.lang.annotation.RetentionPolicy;
+import java.util.Map;
+
+@AutoValue
+public abstract class Board {
+
+
+    public abstract @SerializedName("id") @Nullable String identifier();
```

Comment:
ah, just noticed this, since my IDE autoimported, but there's a missing import for `com.google.gson.annotations.SerializedName`vin this generated source.

Reply:
Great catch! Fixing now